docs/operator/k8s_guide.md

```diff
@@ -0,0 +1,725 @@
+In the current example, a cert for `compass-rucio.cern.ch` (corresponding to the LB `landdb-alias`) was requested:
+
+> [!tip] Certs pro move
+> Create a [grid Host certificate](https://ca.cern.ch/ca/host/Request.aspx?template=ee2host) linked to both servers and auth domains. Use the SAN box add as many domains as you want. Then you'll have one cert to rule them all:
+ 
+![[/img/grid-host-certs.png]]
+
+#### Create the `host`, `key`, `ca` and `GridCA` files
+
+As recommended in the [documentation](https://ca.cern.ch/ca/Help/?kbid=024100), execute:
+```sh
+openssl pkcs12 -in compass-rucio.p12 -clcerts -nokeys -out hostcert.pem
+
+openssl pkcs12 -in compass-rucio.p12 -nocerts -nodes -out hostkey.pem
+
+chmod 400 hostkey.pem 
+chmod 644 hostcert.pem 
+```
+These files have to be stored in the same path as described by the `RAW_SECRETS_SERVERS` variable used in the secrets creation.
+##### Important information about `ca.pem` and `GridCA.pem`
+An additional file, `ca.pem` (i.e. `CERN-bundle.pem`) must be created. 
+This file can be retrieved from lxplus:
+```sh
+cp /etc/pki/tls/certs/CERN-bundle.pem compass-rucio-certs/servers/ca.pem
+```
+The same goes for the `GridCA.pem` file: 
+```sh
+cp /etc/grid-security/certificates/CERN-GridCA.pem compass-rucio-certs/servers/GridCA.pem
+```
+
+### Add the certificates as secrets and mount them on the k8s pods
+Now it is possible to run something similar to the [servers script](https://gitlab.cern.ch/rucio-it/flux-compass/-/blob/master/scripts/rucio-servers-secret.sh?ref_type=heads) to add all the secrets to the cluster.
+
+The output should be of this form: 
+```sh
+Create and apply SERVER secrets
+sealedsecret.bitnami.com/rucio-servers-server-hostcert configured
+sealedsecret.bitnami.com/rucio-servers-server-hostkey configured
+sealedsecret.bitnami.com/rucio-servers-server-cafile configured
+.
+.
+.
+```
+The flux sync can be triggered by ***git pushing*** the content of the `SECRET` folder to the remote repository.
+## Rucio Auth
+[Reference Helm chart](https://gitlab.cern.ch/rucio-it/flux-compass/-/blob/master/sync/rucio-servers-auth.yaml?ref_type=heads).
+
+>The authentication component of Rucio is responsible for securely verifying the identities of users and services attempting to access the Rucio system. It ensures that only authorised entities can perform actions within the system.
+
+As for the servers, four steps are needed to deploy the authentication pod:
+1. Produce the Helm chart
+2. Setup the LB
+3. Produce the certificates related to the landb-alias used
+4. Add the certificates as secrets and mount them on the k8s pods.
+
+### The `httpd` configuration block
+This block maps the certificate to an account name.
+Setting up `x509` auth can be very tricky, please make sure that the [`grid_site_enabled`](https://github.com/rucio/containers/blob/b51bbceb5aab0a1e07d48845f295cbbb175bdcb9/server/rucio.conf.j2#L104) parameter is set on `True`. 
+
+This enables the `auth/x509_proxy` endpoint!
+
```

Comment:
These instructions leave me a more confused than the previous ones. Do we have other pages of rucio-auth documentation we can link here?

Reply:
Not sure I found additional resources on Auth, nevertheless I've added more information to the section to improve the context.